examples/scope/main.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    couchbase-capella = {
+      source = "couchbasecloud/couchbase-capella"
+    }
+  }
+}
+
+provider "couchbase-capella" {
+  authentication_token = var.auth_token
+  host                 = var.host
```

Comment:
The host is only used while testing on different environments, but it is of no use to the customer, so we can remove it